database/migrations/2024_08_08_063713_importers_templates_table.php

```diff
@@ -12,12 +12,12 @@ public function up(): void
     {
         Schema::create('mappers_importers_templates', function (Blueprint $table) {
```

Comment:
@FredPeal since we are not using this table we can remove the content of the import no?